### PR TITLE
Update README & re-enable alpine variants for temurin images

### PR DESCRIPTION
**Pull request by cap10morgan**
Updates the README and re-enables alpine variants for temurin-based images. Followup to #160

### CODE REVIEW

**Comment by cap10morgan** [comment]
This is in response to https://github.com/docker-library/openjdk/pull/495

File: README.md [hunk 1]
```diff
@@ -25,48 +25,54 @@ tools-deps installed.
...
+But you can also specify which distro you'd like by appending it to the end of
+your Docker tag as in the following examples (but note that not every
+combination is provided upstream and thus likewise for us):
 
-JDK 1.8 leiningen on Debian slim-bullseye: `clojure:openjdk-8` or `clojure:openjdk-8-lein` or `clojure:openjdk-8-lein-stretch`
-JDK 1.8 leiningen on Debian buster: `clojure:openjdk-8-buster` or `clojure:openjdk-8-lein-buster`
-JDK 11 tools-deps on Debian slim-bullseye: `clojure:tools-deps` or `clojure:openjdk-11-tools-deps` or `clojure:openjdk-11-tools-deps-slim-bullseye`
+Java 8 leiningen on Debian slim-bullseye: `clojure:openjdk-8` or `clojure:openjdk-8-lein` or `clojure:openjdk-8-lein-bullseye`
+Java 11 leiningen on Debian buster: `clojure:openjdk-11-bullseye` or `clojure:openjdk-11-lein-bullseye`
+Java 17 tools-deps on Debian slim-bullseye: `clojure:tools-deps` or `clojure:temurin-17-tools-deps` or `clojure:temurin-17-tools-deps-focal`
```
Comment (Quantisan) [inline README.md:54]: did you mean "tools-deps on Eclipse Temurin:"?

**Review by Quantisan — APPROVED** [review]
thanks @cap10morgan!

File: README.md (hunk 1 above)
Comment (cap10morgan) [reply README.md:54]: Yes! Good catch!

**Pull request merged**